### PR TITLE
Light v0.8.2

### PR DESCRIPTION
**Pull request by yinweisu**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by yinweisu** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,9 +41,11 @@ requirements:
     - pillow >=9.3,<9.6
     - requests
     - matplotlib-base
-    - ray-default >=2.6.3,<2.7  # [not osx]
     - pydantic >=1.10.4,<2.0
+  {% if variant != "light" %}
```
Comment (giswqs) [inline recipe/meta.yaml:45]: Just curious, where is the `variant` variable defined?

Comment (giswqs) [reply recipe/meta.yaml:45]: I see. Just saw it in another yml.

**Comment by giswqs** [comment]
This is great! I did not know we can use `conda_build_config.yaml` to customize variants.

**Review by giswqs — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (yinweisu) [reply recipe/meta.yaml:45]: Yea. We are not 100% sure if this will work in conda-forge, but worth giving it a shot

**Pull request merged**